react/src/components/Utils/Utils.js

```diff
@@ -241,11 +237,29 @@ export const findFtuUrlById = (ftuPartsArray, searchId) => {
   }
 
   // Find match
-  const foundMatch = ftuPartsArray.find((ftuPart) =>
-    ftuPart.ftu_iri.includes(searchId) || ftuPart.ftu_part_iri.includes(searchId)
+  const foundMatch = ftuPartsArray.find(
+    (ftuPart) =>
+      ftuPart.ftu_iri.includes(searchId) ||
+      ftuPart.ftu_part_iri.includes(searchId),
   );
 
   // Return match digital object URL
   console.log(foundMatch);
   return foundMatch?.ftu_digital_object || null;
 };
+
+// Helper to check if platform is running a variation on MacOS
```

Comment:
Why is this necessary?

Reply:
So we can use MacOS hotkeys for undo/redo